docs/pages/en/ui/asciinema-player/index.md

```diff
@@ -46,39 +46,126 @@ yarn add @nolebase/ui-asciinema
 
 ## Usage
 
-```js
+Add the package to Vite's `ssr.noExternal` configuration. Without this, your site may not build.
+
+```ts
 // vite.config.ts
 export default defineConfig({
-  ssr: {
-    noExternal: [
-      // If there are other packages that need to be processed by Vite, you can add them here.
-      '@nolebase/ui-asciinema',
-    ],
+  vite: {
+    ssr: {
+      noExternal: [
+        "@nolebase/ui-asciinema",
+      ],
+    },
```

Comment:
While it's true that VitePress prefer to use the `vite` property over the `theme/config.ts` configuration file, since `@nolebase/ui-asciinema` is actually a dedicated Vue UI component that doesn't require VitePress as dependencies, I'd suggest that we both keep the original `vite.config.ts` configuration docs as well as the VitePress specific one.

Reply:
Updated.

Recommend changing the "Wrapper of asciinema-player for VitePress documentation sites" branding to something like "Vue wrapper for asciinema-player" ("VitePress-friendly Vue wrapper for asciinema-player" if name checking VitePress is important). I can make that change if you want, but doesn't need to be part of this PR.